pa11y/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+node_modules
```

Comment:
Please remove `node_modules`.

I think it is a leftover from when `pa11y` subfolder had it's own `package.json`.

Reply:
👍